container/overlay/tomcat/conf/catalina.properties

```diff
@@ -191,9 +191,15 @@ rest.api.session.path=ocs/v1/devices/{deviceId}/sessioninfo
 to0.rest.api=${rest.api.server}to0/v1/to0/devices
 
 # SDO IOT Platform OPS onDie-ecdsa Configurations
-org.sdo.ops.ondie-ecdsa-material-path=ondie-ecdsa-material
+org.sdo.ops.ondie-ecdsa-material-path=${catalina.home}/db/ondiecache
 org.sdo.ops.ondie-ecdsa-material-urls=https://tsci.intel.com/content/OnDieCA/certs/,https://tsci.intel.com/content/OnDieCA/crls/
-org.sdo.ops.ondie-ecdsa-material-update=true
+org.sdo.ops.ondie-ecdsa-material-update=false
```

Comment:
Why was this property value changes from true to false?

Reply:
As some of the certs and crls are not production-ready. It was suggested to turn the `**ops.ondie-ecdsa-material-update**` to false. 
Moreover, it helps in keeping **sdo.ops.ondie-ecdsa-material-update** property is in sync with `**sdo.ondiecache.autoupdate**` property.